This is the readme

Ignore test.py please

Python 3 or higher required

If main.py returns file find error
1. enter "readlink -f local_copy.log"
2. Edit line 1 of main.py to reflect that path

The VM im using is very outdated and the python is too. I have been on the struggle bus with mixing compatabilities but I believe it is fully working. I had no problem committing to github or writing the code. This VM is from hell though. My experience in VM's has been Microsoft Azure in TCMG303 which I would give an arm to use over this. I believe my mistake was not getting Ubuntu 18.04 in the intial install which has messed up the entire process.


Also while testing with a smaller amount of lines the code appears to work perfectly. When using the entire log file, the total logs retrieved might be 4 off and the total of 1995 logs appears to be 1 off. This could be due to my method of getting the count being F3'ing "/1995:" and "/1994:"

